reference/5.1/Microsoft.PowerShell.Utility/Compare-Object.md

```diff
@@ -36,6 +36,11 @@ used, (`==`) indicates the value is in both objects.
 If the **reference** or the **difference** objects are null (`$null`), `Compare-Object` generates a
 terminating error.
 
+For simple objects, like strings or numbers, the cmdlet compares the values of the objects. For
+complex objects, you must provide one or more properties to be used for comparison. If you do
+provide a property, the cmdlet checks for **IComparable** interfaces to perform the comparison. If
+that fails, the object are converted to strings and the string values are compared.
```

Comment:
`object` should be plural or changed to 'each object is converted...'

Reply:
Fixed